content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -0,0 +1,130 @@
+    error message.
+
+    A secure way to provide the access token is to create a
+    [GitHub secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+    containing the value of the token, and use the secret in your CI job (see
+    example below).
+
+    ```
+    ...
+      steps:
+        - name: Train model
+          env:
+            STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
+    ...
+    ```
+
+2.  `STUDIO_REPO_URL`: If your repository is on github.com, gitlab.com or
+    bitbucket.org, you do not need to set this environment variable. But if you
+    are using some other Git provider, then you should set the repository url in
+    this format: `{remote-type}:{namespace}/{repo-name}`. For example, for the
+    `example-get-started` repository in the `iterative` namespace,
+    `STUDIO_REPO_URL` should be set to the following value:
+
+    - If you are using a custom (self-hosted) GitLab server, set it to
+      `custom-gitlab:iterative/example-get-started`.
+    - If you are using a GitHub enterprise server, set it to
+      `github:iterative/example-get-started`.
+
+## Send and view live metrics and plots
+
+In the training CI job (which has been configured as detailed above), whenever
+you log your metrics or plots using [DVCLive], they will be automatically sent
+to Iterative Studio. Here is an example of how you can use [DVCLive] in your
+training code:
+
+```
+from dvclive import Live
+
+with Live() as live:
+  for i in range(params["epochs"]):
+    ...
+    live.log_metric("accuracy", accuracy)
+    live.next_step()
+  ...
+  live.end()
+```
+
+Iterative Studio stores the live metrics and plots data in its database.
+
+In the project table, the live metrics are displayed next to the Git commit
+corresponding to the experiment. Updates to the live metrics are highlighted in
+orange.
+
+![](https://static.iterative.ai/img/studio/live_metrics.gif)
+
+Updates to the live metrics are highlighted in orange in the
+[compare pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#compare-experiments)
+as well.
+
+The number of experiments with new updates to the live metrics values are
+displayed in the `Live` icon, which can also be used to filter and show only
+running experiments in the table.
+
+Live plots are displayed in the
+[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots);
+you can see the plots getting populated as Studio receives new updates.
+
+![](https://static.iterative.ai/img/studio/live_plots.gif)
+
+An experiment can have one of the following statuses:
+
+- **Running** - Iterative Studio expects to receive live metrics and plots for
+  these experiments. Note that if the experiment stops due to any error,
+  Iterative Studio will not be aware of this and it will continue to wait for
+  live updates. In this case, you should delete the live metrics row from the
+  project table in Iterative Studio.
```

Comment:
Another live-related Q @daavoo : couldn't it handle errors and auto live.end() the exp?

Reply:
When using live as a context manager, `with Live() as live`, it does handle (Python) errors and exits with `live.end()`.

For other cases like errors outside Python or when using one of the ML Framework integrations it is not possible